api/src/api/v2.js

```diff
@@ -300,25 +303,25 @@ router.get('/runtimes', (req, res) => {
 
 router.get('/packages', async (req, res) => {
     logger.debug('Request to list packages');
-    let packages = await package.get_package_list();
+    let packages = await package_.get_package_list();
 
-    packages = packages.map(pkg => {
```

Comment:
both these `packages` have different types so they should be different variables